components/scream/src/physics/shoc/shoc_functions_f90.cpp

```diff
@@ -1047,7 +1080,7 @@ void update_host_dse_f(Int shcol, Int nlev, Real* thlm, Real* shoc_ql, Real* exn
   std::vector<const Real*> ptr_array = {thlm,  shoc_ql, exner, zt_grid, host_dse};
 
   // Sync to device
-  ekat::host_to_device({phis}, shcol, temp_1d_d);
+  copy_to_device({phis}, shcol, temp_1d_d);
```

Comment:
Follow up on the comment above: here's an example where having to create a vector makes the code a bit less obvious, imho. It'd be easier to parse if we could do
```
view_1d phis_d(shcol);
copy_to_device(phis_d,phis);
```